CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+### 2.2.2
```

Comment:
Can you change to `2.2.1`?

Reply:
Yup I have changed it, looks like there is already a tag in the repo for 2.2.1 though?

Reply:
You are right, I'll fix this in another PR thanks